filter.js

```diff
@@ -153,19 +161,123 @@ const sentEventClick = (eventName) => {
       </div>`;
     };
 
+    const sortList = (arrayList) => {
+      if (sortSettings) {
+        return arrayList.sort((a, b) => a.cost - b.cost);
+      } else {
+        return arrayList.sort(
+          (a, b) =>
+            moment(a.date, "D.MM.YYYY HH:mm").unix() -
```

Comment:
przekmin, że to rzutowanie wykona się przy każdym porównaniu. raczej zależy Ci na parsowaniu  tego jak najmniej razy więc może lepiej zrobić sobie sortList.map(element => ({...element, unixDate: <rzutowana data>}).sort() 

Ciekawostka wtedy -> O(n) razy wykona się parsowanie, a nie O(2n log n)